core/include/detray/propagator/interaction_actors.hpp

```diff
@@ -0,0 +1,26 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "detray/definitions/qualifiers.hpp"
+#include "detray/materials/interaction_kernel.hpp"
+#include "detray/propagator/base_actor.hpp"
+
+namespace detray {
+
+struct pointwise_material_interactor : actor {
+
+    struct state {};
```

Comment:
@niermann999 
I am writing a material interactor inherited from `actor`
I need to access the tracking geometry information (`material_store`) in actor chain but propagation state doesn't have the tracking geometry information. What would be the ideal way to retrieve the geometry info?

Reply:
I think, it might be a good idea to move the detector pointer from the navigator into the navigator state and give it an access call. Then you could do smth like this: ```propagation._navigation.tracking_geometry()```. Would that work for you?